backend/internal/api/handler.go

```diff
@@ -167,6 +170,31 @@ func (h Handler) GetAll(w http.ResponseWriter, r *http.Request) {
 	fmt.Fprint(w, string(allImages))
 }
 
+// getExtension returns extension of given by url image
+func getExtension(bytes []byte) string {
+	if bytes[0] == 0x89 && bytes[1] == 0x50 && bytes[2] == 0x4E && bytes[3] == 0x47 {
+		return "png"
+	}
+	if bytes[0] == 0xFF && bytes[1] == 0xD8 {
+		return "jpg"
+	}
+	if bytes[0] == 0x47 && bytes[1] == 0x49 && bytes[2] == 0x46 && bytes[3] == 0x38 {
+		return "gif"
+	}
+	return ""
+}
+
+// calculateSize returns size of image given in base64
+func calculateSize(imgBase64 string) int {
+	if imgBase64 == "" {
+		return 0
+	} else {
+		l := len(imgBase64)
+		e := strings.Count(imgBase64[len(imgBase64)-2:], "=")
```

Comment:
this method is complex but in `go` it's acceptable but to make this method more understandable you can move this `"="` to the const value - it will help with understanding why are you counting this symbol. In cases like this, when main functionality is based on mathematic skills good practice is to describe the main concept in the comment or paste a link to the article or something